docs-src/guide-rpc-connect.scrbl

```diff
@@ -0,0 +1,519 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[
+  #:version reach-vers
+  #:tag   "guide-rpc-connect"
+  ]{How to connect your language to the Reach RPC server}
+
+It's very easy to create a new a @tech{RPC} client library if one isn't already
+available for your platform. This section describes the necessary steps.
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@section[#:tag "rpc-connect-overview"]{Overview}
+
+As you know from the @seclink["guide-rpc-server-api"]{previous section},
+@tech{RPC} client @tech{frontends} communicate via two channels, named
+@pyin{rpc} and @pyin{rpc_callbacks}, which are obtained by calling
+@pyin{mk_rpc}. Construct these three functions and you'll have already
+completed the majority of what's involved in building a new library.
+
+Your code must also address the following requirements:
+
+@itemlist[
+  @item{Configuring + communicating via @secref{rpc-connect-tls}}
+  @item{Configuring the @secref{rpc-connect-content-type}}
+  @item{Configuring the @secref{rpc-connect-x-api-key}}
+  @item{Selecting the server's @seclink["rpc-connect-host-and-port-envvars"]{host and port}}
+  @item{@seclink["rpc-connect-await-online"]{Waiting for the server to come online}}
+  @item{@seclink["rpc-connect-http2"]{(Optional) use of HTTP/2}}
+]
+
+Since understanding each of these is a prerequisite to building @pyin{rpc} and
+@pyin{rpc_callbacks} we'll begin with them first, then move onto a
+@seclink["rpc-connect-js-lib-walkthrough"]{walkthrough} of how Reach's
+JavaScript @tech{RPC} client library was constructed.
+
```

Comment:
I think it would be a good idea for this section to be a commentary on the implementation of the Python client